lib/broccoli/default-packager.js

```diff
@@ -601,8 +601,9 @@ module.exports = class DefaultPackager {
       });
 
       if (cssMinificationEnabled === true) {
-        preprocessedStyles = preprocessMinifyCss(preprocessedStyles, options);
-        vendorStyles = preprocessMinifyCss(vendorStyles, options);
+        options.minifyCSS.registry = options.registry;
```

Comment:
I think we should tweak `minifyCSS` options in `ember-app`, when they are passed to the constructor

```javascript
let minifyCSS = this.options.minifyCSS;
minifyCSS.registry = this.options.registry;

new DefaultPackager({
  minifyCSS
});
```

Reply:
spread should work in node 6+